rfc6962-bis.xml

```diff
@@ -507,18 +508,11 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
             case consistency_proof: ConsistencyProofDataV2;
             case inclusion_proof: InclusionProofDataV2;
         } data;
-    } TransItemV2;
-
-    struct {
-        Version version;
-        select (version) {
-            case v1: TransItemV1;
-            case v2: TransItemV2;
-        }
-    } TransItem;</artwork>
+    } TransItem;
+          </artwork>
         </figure>
         <t>
-          <spanx style="verb">version</spanx> is the earliest version of this protocol to which the encapsulated data structure conforms. This document is v2. Note that <xref target="RFC6962">v1</xref> did not define <spanx style="verb">TransItem</spanx>, but this document specifies a mechanism (see <xref target="TransItemV1"/>) for v2 implementations to encapsulate existing v1 objects in the <spanx style="verb">TransItem</spanx> structure. Note also that, since each <spanx style="verb">TransItem</spanx> object is individually versioned, future revisions to this protocol could conceivably update some encapsulated data structures without having to update all of them.
+          <spanx style="verb">version</spanx> is the earliest version of this protocol to which the encapsulated data structure conforms. This document is v2. Note that <xref target="RFC6962">v1</xref> did not define <spanx style="verb">TransItem</spanx>, but this document provides guidelines (see <xref target="v1_coexistence"/>) on how v2 implementations can co-exist with v1 implementations. Note also that, since each <spanx style="verb">TransItem</spanx> object is individually versioned, future revisions to this protocol could conceivably update some encapsulated data structures without having to update all of them.
```

Comment:
I realise this is pre-existing text, but I now find it confusing. If, for example, I added a new TransType, it would seem I would been to bump the version field ... however, that doesn't actually seem helpful.